actor-typed-tests/src/test/scala/org/apache/pekko/actor/typed/DeadLetterSpec.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.pekko.actor.typed
+
+import org.apache.pekko.actor.IllegalActorStateException
+import org.apache.pekko.actor.testkit.typed.scaladsl.{ LogCapturing, ScalaTestWithActorTestKit }
+import org.apache.pekko.actor.typed.scaladsl.AskPattern.{ schedulerFromActorSystem, Askable }
+import org.apache.pekko.actor.typed.scaladsl.Behaviors
+import org.apache.pekko.actor.typed.scaladsl.Behaviors._
+import org.apache.pekko.util.Timeout
```

Comment:
@mdedetrich How to format these imports as the style your suggested, is there any sbt command?

Reply:
It appears that the contribution guide mentions this, but `sortImports` sort too many files... that not correct behavior.
https://github.com/apache/incubator-pekko/blob/main/CONTRIBUTING.md#scala-style